src/test/java/core/basesyntax/fruitoperation/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.fruitoperation.strategy;
+
+import core.basesyntax.fruitoperation.Operation;
+import core.basesyntax.fruitoperation.OperationBalance;
+import core.basesyntax.fruitoperation.OperationPurchase;
+import core.basesyntax.fruitoperation.OperationReturn;
+import core.basesyntax.fruitoperation.OperationSupply;
+import core.basesyntax.fruitoperation.Operations;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.FruitServiceImpl;
+import core.basesyntax.service.file.DataReader;
+import core.basesyntax.service.file.impl.FileReader;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
+    private static FruitService fruitService;
+
+    @Before
+    public void setUp() throws Exception {
+        Map<Operations, Operation> operationMap = new HashMap<>();
+        operationMap.put(Operations.B, new OperationBalance());
+        operationMap.put(Operations.P, new OperationPurchase());
+        operationMap.put(Operations.R, new OperationReturn());
+        operationMap.put(Operations.S, new OperationSupply());
+        OperationStrategy operationStrategy = new OperationStrategyImpl(operationMap);
+        fruitService = new FruitServiceImpl(operationStrategy);
+    }
+
+    @Test
+    public void test5ReportFormatting_twoBalanced() {
+        try {
+            DataReader dataReader = new FileReader();
+            fruitService.saveToStorage(
+                    dataReader.readData("src/main/resources/test5_incorrect.csv"));
```

Comment:
```suggestion
                    dataReader.readData("src/test/resources/test5_incorrect.csv"));
```
same for other files that you use in tests

Reply:
done